### PR TITLE
Added the Spanish translation

### PR DESCRIPTION
**Pull request by NefixEstrada**
I've added the Spanish translation. Maybe some words doesn't make
sense because of the missing context of the sentences.

Fixing #18

### CODE REVIEW

File: i18n/bot.es-es.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    Para enviar un fichero a tu Kindle, haz click a <b>Enviar fichero</b>
+    Para cambiar tu correo electrónico, haz click a <b>Establecer correo electrónico</b>
+  startnewuser: |
+    ¡Hola!
+    Éste es un bot que envía ficheros a tu Kindle.
+    Para empezar, escribe tu correo electrónico Kindle.
+    Debe terminar en <code>kindle.com</code>
+  startolduser: |
+    <b>¡Bienvenido de nuevo</b>!
+    Your registered e-mails are:
```
Comment (albertocc) [inline i18n/bot.es-es.yml]: You forgot to translate this line into Spanish

Comment (NefixEstrada) [reply i18n/bot.es-es.yml]: Woops

**Comment by GabrielRF** [comment]
@NefixEstrada Could you please talk to me on Telegram? https://t.me/GabrielRF

**Comment by falbala84** [comment]
There are **ortographic errors** in this file. I already corrected it some days ago but I think my changes were dismissed. 

- éste --> This is always WRONG. It MUST be "este" without accent mark

> ¿Qu**e** hace **é**ste bot? 

Should be ---> ¿Qu**é** hace **e**ste bot?  (Watchout the accent marks)

> Éste bot puede enviar ficheros a tu Kindle cómo si los estuvieras enviando por correo electrónico.

--> should be: "**E**ste bot puede enviar ficheros a tu Kindle c**o**mo si los estuvieras enviando por correo electrónico."

> Se**ss**ión 

--> should be "se**s**ión"

> **É**ste bot está en desarrollo constante!

It should be: "**¡E**ste bot está en desarrollo constante!

**Pull request merged**